xbmc/windowing/X11/WinEventsX11.cpp

```diff
@@ -312,6 +312,12 @@ bool CWinEventsX11::MessagePump()
       if (xevent.xgeneric.serial == serial)
         continue;
 
+      if (g_application.IsDPMSActive())
+      {
+        CLog::Log(LOGDEBUG,"CWinEventsX11::MessagePump - skip XRRNotifyEvent since DPMS is active");
```

Comment:
Is this going to end up flooding the log?

Reply:
When I tested it in debug mode there was no flooding just a single log entry when the monitor is going into suspend.